src/templates/article.js

```diff
@@ -9,12 +9,48 @@ import ArticleShareFooter from '../components/dev-hub/article-share-footer';
 import BlogPostTitleArea from '../components/dev-hub/blog-post-title-area';
 import Layout from '../components/dev-hub/layout';
 import RelatedArticles from '../components/dev-hub/related-articles';
-import { size } from '../components/dev-hub/theme';
+import { screenSize, size } from '../components/dev-hub/theme';
 import Series from '../components/dev-hub/series';
 import { getTagLinksFromMeta } from '../utils/get-tag-links-from-meta';
 import { getTagPageUriComponent } from '../utils/get-tag-page-uri-component';
-import { normalizePath } from '../utils/normalize-path';
 import { useSiteMetadata } from '../hooks/use-site-metadata';
+import ShareMenu from '../components/dev-hub/share-menu';
+import ContentsMenu from '../components/dev-hub/contents-menu';
+import Contents from '../../src/components/Contents.js';
+import { getNestedValue } from '../utils/get-nested-value';
+
+const findSectionHeadings = (nodes, key, value) => {
```

Comment:
Does anybody know if its possible to have a heading within a heading with our rsts?